fedmsg/tests/fedmsg-test-config.py

```diff
@@ -26,11 +26,12 @@
 here = os.getcwd()
 hostname = socket.gethostname().split('.', 1)[0]
 
-ssl_enabled_for_tests = True
+import imp
```

Comment:
``imp`` is actually a deprecated module since 3.4 so we shouldn't use it (even though we're not quite Python 3 compatible yet, but soon!). In addition, I don't know of a good reason to _not_ check for a module's presence by attempting to import it.